### PR TITLE
Some additional methods are required when subclassing.

### PR DESCRIPTION
**Pull request by kenguest**
Required for a subclass of the client.

### CODE REVIEW

File: OpenIDConnectClient.php [hunk 1]
```diff
@@ -417,7 +417,7 @@ public function getRedirectURL() {
               ?: @$_SERVER['SERVER_ADDR'];
 
         // Don't include port if it's 80 or 443 and the protocol matches
-        $port = (443 == $port) || (80 == $port) ? '' : ':' . $port;
```
Comment (jumbojett) [inline OpenIDConnectClient.php]: @kenguest could you restore line 420? This is a recent bug fix.

File: OpenIDConnectClient.php [hunk 2]
```diff
@@ -417,7 +417,7 @@ public function getRedirectURL() {
...
+        $port = ($protocol === 'https' && $port === 443) || ($protocol === 'http' && $port === 80) ? '' : ':' . $port;
```
Comment (kenguest) [inline OpenIDConnectClient.php]: @jumbojett Is this the correct version? It's in there now.

Comment (jumbojett) [inline OpenIDConnectClient.php]: It looks like it's behind one commit. I committed a fix to master a few days ago. No need to fiddle with git commands. Just change this one line and you should be set. 👍 

https://github.com/jumbojett/OpenID-Connect-PHP/commit/fcdf8635a60d31460700336e1abd905762bf41c9

Comment (kenguest) [inline OpenIDConnectClient.php]: The comment says the protocol has to match - should I keep things sane and remove the comment too? :)

Comment (kenguest) [inline OpenIDConnectClient.php]: Done. Sanity isn't over-rated. :-)

**Comment by jumbojett** [comment]
@rasodu this looks good to me. Are we ready to accept this PR?

**Comment by rasodu** [comment]
Looks good. I am going to mark first item in #57 complete.

**Pull request merged**